Foundation/URLSession/http/HTTPURLProtocol.swift

```diff
@@ -368,395 +368,9 @@ fileprivate extension _HTTPURLProtocol._Body {
         }
     }
 }
-
-fileprivate func errorCode(fileSystemError error: Error) -> Int {
-    func fromCocoaErrorCode(_ code: Int) -> Int {
-        switch code {
-        case CocoaError.fileReadNoSuchFile.rawValue:
-            return NSURLErrorFileDoesNotExist
-        case CocoaError.fileReadNoPermission.rawValue:
-            return NSURLErrorNoPermissionsToReadFile
-        default:
-            return NSURLErrorUnknown
-        }
-    }
-    switch error {
-    case let e as NSError where e.domain == NSCocoaErrorDomain:
-        return fromCocoaErrorCode(e.code)
-    default:
-        return NSURLErrorUnknown
-    }
-}
-
-internal extension _HTTPURLProtocol {
-    /// The data drain.
-    ///
-    /// This depends on what the delegate / completion handler need.
```

Comment:
This doc comment is missing in the corresponding new version.

Reply:
Addressed the comments